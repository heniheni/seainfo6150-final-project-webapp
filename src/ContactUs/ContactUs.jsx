import React from "react";
import styles from "./ContactUs.module.css";

const ContactUs = (props) => {
  //const image1 = require(props.team.teamLogoUrl);
  return (
    <div className={styles.mainContainer}>
      <div>
        <h1>Contact Us</h1>
      </div>

      <hr />

      <div>
        <h3 className={styles.subHeading}>Contact Details</h3>
      </div>
<br/>
      <div className={styles.content}>
        <p>
          BCCI-IPL
          <br />
          4th Floor, Cricket Centre
          <br />
          Wankhede Stadium
          <br />
          ‘D’ Road, Churchgate
          <br />
          Mumbai- 400020
          <br />
          India
          <br /><br/>
          T +91 22 22800300
          <br />
          T +91 22 61580300
          <br />
          F +91 22 22800354
          <br />
        </p>
      </div>
    </div>
  );
};

export default ContactUs;
