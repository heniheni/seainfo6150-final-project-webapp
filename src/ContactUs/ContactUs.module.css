.mainContainer {
  display: block;
  box-shadow: 4px 3px 8px 1px grey;
  width: 60%;
  padding: 20px;
  margin-left: auto;
  margin-right: auto;
  margin-top: 10%;
  margin-bottom: 20%;
}
.stylingBreadCrumb{
  text-decoration: none;
  font-weight: bold;
}
