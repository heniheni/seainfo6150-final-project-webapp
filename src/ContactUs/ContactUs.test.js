import React from "react";
import ContactUs from "./ContactUs";


describe("ContactUs tests", () => {

    it("renders correctly", () => {
      const { container } = render(<ContactUs />);
      expect(container).toMatchSnapshot();
    });
  });