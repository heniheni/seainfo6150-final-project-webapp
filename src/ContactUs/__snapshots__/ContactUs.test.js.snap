// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContactUs tests renders correctly 1`] = `
<div>
  <div
    class="mainContainer"
  >
    <div>
      <h1>
        Contact Us
      </h1>
    </div>
    <hr />
    <div>
      <h3
        class="subHeading"
      >
        Contact Details
      </h3>
    </div>
    <br />
    <div
      class="content"
    >
      <p>
        BCCI-IPL
        <br />
        4th Floor, Cricket Centre
        <br />
        Wankhede Stadium
        <br />
        ‘D’ Road, Churchgate
        <br />
        Mumbai- 400020
        <br />
        India
        <br />
        <br />
        T +91 22 22800300
        <br />
        T +91 22 61580300
        <br />
        F +91 22 22800354
        <br />
      </p>
    </div>
  </div>
</div>
`;
